app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php

```diff
@@ -2,11 +2,8 @@
 
 class CustomGento_ConfigurableTierPrices_Helper_Data extends Mage_Core_Helper_Abstract
 {
-
     const XML_PATH_IS_ENABLED = 'sales/customgento_configurabletierprices/is_enabled';
     const XML_PATH_DISABLED_FOR_CATEGORY = 'sales/customgento_configurabletierprices/disabled_for_category';
-
-    // attribute code length must be less than 30 symbols!
```

Comment:
Please keep this comment (as it is non-intuitive to use an abbreviated version of the extension name here).